src/traits/base.php

```diff
@@ -135,6 +135,15 @@ protected function mock() {
 		return $this->mock;
 	}
 
+	/**
+	 * Things needed to run before the test class performs tests.
+	 *
+	 * @return void
+	 */
+	public function setUpBeforeClass(): void { // phpcs:ignore
+		remove_action( 'init', 'twentytwentyfive_register_block_bindings' );
+	}
```

Comment:
Add a call to the parent method, otherwise we're sure to break something in the WordPress test framework.